test/stubs/contentful/get-entries-campaign-with-overrides.json

```diff
@@ -0,0 +1,40 @@
+{
+  "sys": {
+    "type": "Array"
+  },
+  "total": 1,
+  "skip": 0,
+  "limit": 100,
+  "items": [
+    {
+      "sys": {
+        "space": {
+          "sys": {
+            "type": "Link",
+            "linkType": "Space",
+            "id": "owik07lyerdj"
+          }
+        },
+        "id": "6tlsrM2RTa40oMqwcIKSg6",
+        "type": "Entry",
+        "createdAt": "2017-02-15T20:30:31.426Z",
+        "updatedAt": "2017-05-12T15:20:14.634Z",
+        "revision": 10,
+        "contentType": {
+          "sys": {
+            "type": "Link",
+            "linkType": "ContentType",
+            "id": "campaign"
+          }
+        },
+        "locale": "en-US"
+      },
+      "fields": {
+        "campaignId": "7",
+        "gambitSignupMenuMessage": "Great! Little moments can have a big impact. Complimenting someone can turn their whole day around.\n\nDo this: post encouraging notes in places that can trigger low self-esteem, like school bathrooms.\n\nThen, text START to share a photo of the messages you posted (and you'll be entered to win a $1500 scholarship)! \n\nBonus: Text back a photo of your posted note by 11:59PM tonight and you can enter to win a gift card of your choice! (Winners announced this week). ",
```

Comment:
The `text START` text will be one of those instances where if we ever change the Start Reportback Command config variable, these tests would break. Nothing we need to change for merging, but it does make me wonder if the stubbed objects should be returned by a function, where we can pass a parameter like the command reportback to render. (or the campaign ID to use as I've mentioned before).

There's definitely something to be said how it's convenient to just store the stub data as a json file too. Just food for thought 🍔 .